src/Database/Log/LoggingStatement.php

```diff
@@ -109,15 +109,37 @@ public function bindValue($column, $value, $type = 'string')
      * Sets the logger object instance. When called with no arguments
      * it returns the currently setup logger instance
      *
+     * @deprecated 3.5.0 Use getLogger() and setLogger() instead.
```

Comment:
I really don't care, my point is that it is a common PITA that one person tells you to do it this way, then somebody else comes in tells you the opposite. So I would like to get phpcs to detect it and enforce you to do it right or at least a wiki page for reference to look it up.